traitsui/testing/tester/ui_wrapper.py

```diff
@@ -161,6 +161,24 @@ def _perform_or_inspect(self, interaction):
                 with _event_processed():
                     return handler(self, interaction)
 
+        try:
+            # there may not be a handler registered for this interaction on
+            # the original target.  Instead, it may be registered for DefaultTarget
```

Comment:
Thank you for adding these explanations. However having so many comments beg a different question: Is the code block difficult to understand? If so, we might be better off extracting this nested-try-block into a separate (private) method so that we can better document it.